src/PageContent/Calendar/ContactContent.js

```diff
@@ -0,0 +1,163 @@
+import React, { Component } from 'react';
+import { AccountType, StringFormat } from '../../Enums';
+import './Content.css';
+import phone from '../../icons/phone.svg';
+
+class ContactContent extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            edit: false,
+            contact: {},
+            memberList: []
+        };
+        if (this.props.accountType === AccountType.DONATING_AGENCY_MEMBER) {
+            this.state.contact = this.props.delivery.donatingAgency.contact;
+            this.state.memberList = this.props.delivery.donatingAgency.contact.memberList;
+        } else if (this.props.accountType === AccountType.RECEIVING_AGENCY) {
+            this.state.contact = this.props.delivery.receivingAgency.contact;
+        }
+
+        this.edit = this.edit.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+        this.handleChangeDA = this.handleChangeDA.bind(this);
+    }
+    edit() {
+        this.setState({
+            edit: true
+        });
+    }
+
+    handleChange(e) {
+        e.preventDefault();
+        this.setState({
+            name: e.target.name.value,
+            phone: e.target.phone.value,
+            edit: false
+        });
+    }
+
+    handleChangeDA(e) {
+        e.preventDefault();
+        let index = e.target.primaryContact.value;
+        let memberName = this.state.memberList[index].name;
+        let memberPhone = this.state.memberList[index].phone;
+        this.setState({
```

Comment:
same here, should just modify `contact` that's already in the `state` instead of assigning new fields:
```
this.setState({
    contact: {
        name: memberName,
        phone: memberPhone,
        email: memberEmail
    }
});
```